packit_service/worker/handlers/bodhi.py

```diff
@@ -192,6 +192,10 @@ def run(self) -> TaskResults:
                 target_model.set_status("error")
                 target_model.set_data({"error": error})
 
+            except Exception:
+                target_model.set_status("error")
```

Comment:
could we also do the `target_model.set_data({"error": error})`?

Reply:
`error` being the exception and traceback? So only storing it in the DB, not creating a user-facing issue?

Reply:
yes, I meant the exception and traceback. But if it is the case of our unexpected bug, maybe we could also mark that in the `error`, something like `Internal error, please contact us`, and we could also prioritise #2426 for that.

Reply:
and as for creating user-facing issue, I would do the same, like a message about `Internal error`, but we need to account for the retries the same way as in the `PackitException` branch to avoid duplicated issues/comments